myhpom/urls.py

```diff
@@ -25,4 +25,5 @@
     url(r'^profile/?$', myhpom_views.view_profile, name='view_profile'),
     url(r'^profile/edit/?$', myhpom_views.edit_profile, name='edit_profile'),
     url(r'^faq/?$', myhpom_views.faq, name='faq'),
+    url(r'^why-plan/?$', myhpom_views.why_plan, name='why-plan'),
```

Comment:
Would it make sense to generalize this a little? Have a single url definition for content, with the template name as the parameter. See also comment on the view.